src/applications/vaos/tests/fixtures/MockReferralDraftAppointmentResponse.js

```diff
@@ -269,7 +270,7 @@ class MockReferralDraftAppointmentResponse {
     // Return complete response matching the expected format
     return {
       data: {
-        id: 'EEKoGzEf',
+        id: referralNumber,
```

Comment:
I don't think this should be the referralNumber but I could be wrong. Referral number is different from the id of the draft appointment.